package.json

```diff
@@ -59,13 +59,14 @@
     "uuid": "^3.0.1"
   },
   "devDependencies": {
+    "@types/node": "^7.0.18",
```

Comment:
Again, I don't know much about TS. I just had a look at the package on npm. Does this need to be a dev dependency on this module for the definitions to work? I would have thought anyone using TS would have the node definitions installed somewhere else?